pkg/settings/setting.go

```diff
@@ -155,6 +155,9 @@ var (
 	// UIBrand High level 'brand' value, for example `suse`.
 	UIBrand = NewSetting("ui-brand", "")
 
+	// Fallback env, not a user-facing setting, used to indicate if this is a Prime install
+	PrimeUIBrand = NewSetting("base-ui-brand", os.Getenv("BASE_UI_BRAND"))
```

Comment:
the getEnv check is correct, but this should be on the `ui-brand` setting, so we still use branding through that

Reply:
ah, ok.  I slightly misunderstood you.  Will fix it up